Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderDataSystem.h

```diff
@@ -22,9 +24,14 @@ namespace ScriptCanvasEditor
 
 namespace ScriptCanvasBuilder
 {
-    class DataSystem
-        : public DataSystemRequestsBus::Handler
+// #define DATA_SYSTEM_STATUS(window, msg, ...) AZ_TracePrintf(window, msg, __VA_ARGS__);
```

Comment:
This defined diagnostic messaging for the data system which will soon be routed to imgui.